vega_sim/vegahome/genesis.json

```diff
@@ -179,7 +179,7 @@
       "spam.pow.numberOfPastBlocks": "100",
       "spam.pow.numberOfTxPerBlock": "1000",
       "spam.protection.delegation.min.tokens": "100000000000000000",
-      "spam.protection.max.batchSize": "30",
+      "spam.protection.max.batchSize": "200",
```

Comment:
Not sure about how I feel moving away from `mainnet` values here.

Maybe a separate genesis for sim works now devops are using market-sim bots?